.pipelines/templates/expand-compliance.yml

```
@@ -0,0 +1,12 @@
+# This Yaml Document has been converted by ESAI Yaml Pipeline Conversion Tool.
+# Please make sure to check all the converted content, it is your team's responsibility to make sure that the pipeline is still valid and functions as expected.
+parameters:
+  architecture: x86
+  version: 6.2.0
+steps:
+- powershell: |
+    Expand-Archive -Path "$(System.ArtifactsDirectory)\results\PowerShell-${{ parameters.version }}-symbols-win-${{ parameters.architecture }}.zip" -Destination "$(Build.StagingDirectory)\symbols\${{ parameters.architecture }}"
+  displayName: Expand symbols zip - ${{ parameters.architecture }}
+- powershell: |
+    tools/releaseBuild/createComplianceFolder.ps1 -ArtifactFolder "$(Build.StagingDirectory)\symbols\${{ parameters.architecture }}" -VSTSVariableName 'CompliancePath'
+  displayName: Expand Compliance file - ${{ parameters.architecture }}
```

```suggestion
  displayName: Expand Compliance file - ${{ parameters.architecture }}

```
add posix file ending

---

fixed